anthos-bm-gcp-terraform/modules/init/variables.tf

```diff
@@ -106,3 +112,8 @@ variable "priv_key_path_template" {
   type        = string
   default     = "../../resources/.ssh-key-%s.priv"
 }
+
+variable "local_files_ready" {
```

Comment:
Let's remove this and see if we can enforce depends on directly on the `init` module in main.tf

Reply:
I was originally having difficulty as the `init` module contains sub-modules that utilize `count`s that aren't evaluated till apply, but I think I have a new idea how to do this more cleanly.